client/Components/Map.js

```diff
@@ -8,21 +8,83 @@ const Wrapper = styled.div`
     height: 320px;
   }
 
+  .marker {
+    width: 10px;
+    height: 10px;
+    background-color: #000;
+    border: #000 4px solid;
+    border-radius: 50%;
+    opacity: 0.99;
+  }
+
+  .marker.nearby {
+    background-color: transparent;
+    border: blue 4px solid;
+  }
+
+  .building-link {
+    line-height: 20px;
+    height: 100%;
+    width: 137.8px;
+    color: blue;
+    font-size: 16px;
+    text-align: center;
+    text-decoration: none;
+    appearance: none;
+    background: none;
+    border: 0;
+    padding: 0;
+    outline: none;
+  }
+
+  .building-link:hover {
+    text-decoration: underline
+  }
 `
 
 const Map = ({locationData}) => {
 
-  const coordinates = [locationData.coordinates[1], locationData.coordinates[0]]
+  const { origin, nearbyWorkspaces } = locationData;
+  const coordinates = origin.geometry.coordinates;
```

Comment:
Glad to see using geometry worked out. I find the [long, lat] system a little counter-intuitive